--- docker-entrypoint.orig.sh	2021-12-20 18:12:21.406996532 +0100
+++ docker-entrypoint.sh	2021-12-21 13:01:54.384229555 +0100
@@ -11,7 +11,10 @@
 set -eu
 
 if [ ! -d "/run/clamav" ]; then
-	install -d -g "clamav" -m 775 -o "clamav" "/run/clamav"
+	install -d -g "clamav" -m 775 -o "root" "/run/clamav"
+else
+  chown root:clamav /run/clamav
+  chmod 775 /run/clamav
 fi
 
 # Assign ownership to the database directory, just in case it is a mounted volume
@@ -76,6 +79,8 @@
 		clamav-milter &
 	fi
 
+	/usr/local/sbin/update-other-rules.sh &
+
 	# Wait forever (or until canceled)
 	exec tail -f "/dev/null"
 fi
